src/toil/cwl/cwltoil.py

```diff
@@ -2948,8 +2964,39 @@ def makeRootJob(
     :return:
     """
     if options.run_imports_on_workers:
+        filenames = extract_workflow_inputs(options, initialized_job_order, tool)
+        files_to_data = get_file_sizes(
+            filenames, toil._jobStore, include_remote_files=options.reference_inputs
+        )
+
+        # files with a associated filesize that are valid to be imported on workers
+        valid_files_to_data = dict()
```

Comment:
This isn't *really* mapping to file data (bits in the file), right? This is metadata consisting of file size and... something.

Maybe this should be renamed and have a comment explaining what's actually in the values? Or an inline type hint and not just auto-typing?